packages/node/src/policy-gen/policy-gen-compartment-map.js

```diff
@@ -60,8 +62,16 @@ export const loadCompartmentMap = async (
    * resource key in policy. We have no use for Endo's label field, so we
    * replace labels with canonicalNames derived from path.
    */
-  values(nodeCompartmentMap.compartments).map((compartmentDescriptor) => {
-    compartmentDescriptor.label = getCanonicalName(compartmentDescriptor)
+  values(nodeCompartmentMap.compartments).forEach((compartmentDescriptor) => {
+    if (compartmentDescriptor.name === ATTENUATORS_COMPARTMENT) {
```

Comment:
Why do we check it here?

Reply:
because you told me to check it here 😉 

your reasoning was that the implementation of `getCanonicalName` should not know anything about `ATTENUATORS_COMPARTMENT`.  

while it's unlikely that we'll _ever_ hit this branch, this is a case of "making assertions about impossible things"

- [ ] add a `/* c8 ignore next */` here, since this "should never happen"